site/docs/v1/tech/lambdas/sony-playstation-network-reconcile.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: doc
+title: Sony Playstation Network Reconcile
+description: An overview of the Sony Playstation Network Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Sony Playstation Network Reconcile lambda
+
+When an Sony Playstation Network identity provider is used to complete a federated login request FusionAuth will use the email address to reconcile user. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
+
+When you create a new lambda using the FusionAuth UI we will provide you an empty function for you to implement.
+
+If you are using the API to create the lambda you will need to ensure your function has the following signature:
+
+[source,javascript]
+----
+function reconcile(user, registration, idToken) {
+  // Lambda code goes here
+}
+----
+
+This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
+
+* `user` - the FusionAuth User object
+* `registration` - the FusionAuth UserRegistration object
+* `idToken` - the JSON payload returned by the Sony Playstation Network Token Info API
+// TODO 👆 check above for accuracy
```

Comment:
This varies by IdP, but maybe we can just always call it `idToken`.  In fact, I'm not sure why we send back the token that we do.  Sometimes we make 2 API calls, but only return 1 token/json.  See here [lines 63-74](https://github.com/FusionAuth/fusionauth-api/blob/1.28.0/src/main/java/io/fusionauth/api/service/authentication/SonyPSNIdentityProviderAuthenticationService.java#L63-L74) in the `SonyPSNIdentityProviderAuthenticationService`.